src/modules/flow/types/Nodes/GeminiChatNode.ts

```
@@ -0,0 +1,212 @@
+      }
+
+      try {
+        console.log(`Executing Gemini Chat node ${n.id} with model: ${model}`);
+        console.log(
+          `Prompt: "${prompt.substring(
+            0,
+            50
+          )}..." | System: "${system.substring(0, 50)}..."`
+        );
+
+        const body: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)